main/tests/cypress/cypress/integration/create-project/create_project.spec.js

```diff
@@ -26,89 +26,168 @@ describe(__filename, function () {
 
     // then ensure we are on the preview page
     cy.get('.create-project-ui-panel').contains('Configure Parsing Options');
-  });
 
-  it('Test project naming', function () {
+    // preview and click next
+    cy.doCreateProjectThroughUserInterface();
+  });
+  it('Test the create project from this computer based on TSV', function () {
+    // navigate to the create page
     cy.visitOpenRefine();
-    cy.createProjectThroughUserInterface('food.mini.csv');
-    cy.get('.create-project-ui-panel').contains('Configure Parsing Options');
+    cy.navigateTo('Create Project');
+    cy.get('#create-project-ui-source-selection-tabs > div')
+      .contains('This Computer')
+      .click();
+    cy.get('.create-project-ui-source-selection-tab-body.selected').contains(
+      'Locate one or more files on your computer to upload'
+    );
+    // add file
+    const tsvFile = {
+      filePath: 'param.tsv',
+      mimeType: 'text/tab-separated-values',
+    };
+    cy.get(
+      '.create-project-ui-source-selection-tab-body.selected input[type="file"]'
+    ).attachFile(tsvFile);
+    cy.get(
+      '.create-project-ui-source-selection-tab-body.selected button.button-primary'
+    ).click();
     cy.get(
       '.default-importing-wizard-header input[bind="projectNameInput"]'
-    ).type('this is a test');
-    cy.doCreateProjectThroughUserInterface();
+    ).should('have.value', 'param tsv');
 
-    cy.get('#project-name-button').contains('this is a test');
-  });
+    // then ensure we are on the preview page
+    cy.get('.create-project-ui-panel').contains('Configure Parsing Options');
 
-  it('Test project tagging', function () {
+    // preview and click next
+    cy.doCreateProjectThroughUserInterface();
+  });
+  it('Test the create project from clipboard based on CSV', function () {
+    // navigate to the create page
     cy.visitOpenRefine();
-    cy.createProjectThroughUserInterface('food.mini.csv');
+    cy.navigateTo('Create Project');
+    cy.get('#create-project-ui-source-selection-tabs > div')
+      .contains('Clipboard')
+      .click();
+    cy.get('#or-import-clipboard').should(
+      'to.contain',
+      'Paste data from clipboard here:'
+    );
+    // add file
+    const csvFile = `Username; Identifier;First name;Last name
+    booker12;9012;Rachel;Booker
+    grey07;2070;Laura;Grey
+    johnson81;4081;Craig;Johnson
+    jenkins46;9346;Mary;Jenkins
+    smith79;5079;Jamie;Smith`;
+    cy.get('textarea').invoke('val', csvFile);
```

Comment:
Why not using `cy.get('.selector').type(csvFile);` ?

Reply:
Invoke is more like copy-paste rather than the type which is individually typing each character. That's why I used invoke to have tested more resembling to copy-paste in clipboard.

Reply:
Agree, good with it